test.py

```diff
@@ -1,4 +1,5 @@
 from test1 import func
```

Comment:
There is no test to verify the functionality of `func`. The code only prints strings. Add assertions to actually test the imported function `func` from `test1.py`.

```suggestion
assert func(1) == expected_value, "Test case failed"
```